### PR TITLE
add docs for adding TSP and TDL data

### PR DESCRIPTION
**Pull request by mikena-truss**
## Description

During the TDL rate/score import, I hit a few snags with missing TDL and TSP data. Here are some notes on how I imported that data.

## Code Review Verification Steps

* [ ] The requirements listed in
 [Querying the Database Safely](./docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](./docs/database.md#zero-downtime-migrations))
  * [ ] Secure migrations have been tested using `scripts/run-prod-migrations`

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/165126401) is the original story

### CODE REVIEW

File: docs/data/tspp-data-creation.md [hunk 1]
```diff
@@ -175,6 +175,52 @@ AND
...
+INSERT INTO temp_tdls (id, source_rate_area, destination_region, code_of_service, created_at, updated_at, import)
+VALUES
+  (uuid_generate_v4(), 'US4965500', '1', '2', now(), now(), true),
+  (uuid_generate_v4(), 'US4965500', '1', 'D', now(), now(), true),
+  /* ... */
+  (uuid_generate_v4(), 'US4965500', '10', '2', now(), now(), true);
+```
+
+This will add the new entries to the temporary TDL table,
+forcing them to adhere to adhere to any table constraints
```
Comment (kimallen) [inline docs/data/tspp-data-creation.md]: 'to adhere' twice

File: docs/data/tspp-data-creation.md [hunk 2]
```diff
@@ -194,6 +240,43 @@ WHERE
   tsd.scac = tsp.standard_carrier_alpha_code;
 ```
 
+Similar to TDLs,
+there may be missing TSPs.
+Currently, we're not using any of this TSP data for production moves,
+but we have to satisfie the foreign key constraints for the TSPP data.
```
Comment (kimallen) [inline docs/data/tspp-data-creation.md]: satisfie --> satisfy

File: docs/data/tspp-data-creation.md [hunk 3]
```diff
@@ -175,6 +175,52 @@ AND
...
+  (uuid_generate_v4(), 'US4965500', '1', 'D', now(), now(), true),
+  /* ... */
+  (uuid_generate_v4(), 'US4965500', '10', '2', now(), now(), true);
+```
+
+This will add the new entries to the temporary TDL table,
+forcing them to adhere to adhere to any table constraints
+and generating new UUIDs to be consistent across environments.
+
+We'll now create a new migration with that data (replace your migration filename:
```
Comment (kimallen) [inline docs/data/tspp-data-creation.md]: missing closing ')'?

File: docs/data/tspp-data-creation.md [hunk 4]
```diff
@@ -175,6 +175,52 @@ AND
...
+
+```bash
+./bin/soda generate sql add_new_scacs
+rm migrations/20190410152949_add_new_tdls.down.sql
```
Comment (kimallen) [inline docs/data/tspp-data-creation.md]: for the sake of example, should the names be the same for what is removed?  right now it's generating add_new_scacs and removing add_new_tdls

File: docs/data/tspp-data-creation.md [hunk 5]
```diff
@@ -175,6 +175,52 @@ AND
...
+  /* ... */
+  (uuid_generate_v4(), 'US4965500', '10', '2', now(), now(), true);
+```
+
+This will add the new entries to the temporary TDL table,
+forcing them to adhere to adhere to any table constraints
+and generating new UUIDs to be consistent across environments.
+
+We'll now create a new migration with that data (replace your migration filename:
+
```
Comment (kimallen) [inline docs/data/tspp-data-creation.md]: Can you summarize what the below code is doing in plain English?  It's a little hard for me to follow what is happening

**Comment by mikena-truss** [comment]
@kimallen I think I addressed your comments. There was a small explanation for the TDL import below the code, but I added a little more.
